autospec/pkg_integrity.py

```diff
@@ -113,7 +113,9 @@ def verify(self, _, tarfile, signature):
             # sig file may be ascii-armored, so dearmor it first...
             args = self.args + ['--dearmor', '--output', '-', signature]
             output, err, code = self.exec_cmd(args)
-            if code != 0:
+            if code == 2 and err.decode('utf-8').startswith('gpg: no valid OpenPGP data found.'):
+                util.print_warning("Ignoring: {}".format(err.decode('utf-8')))
+            elif code != 0:
```

Comment:
Rather than decoding the err twice, mind just doing a `errmsg = err.decode('utf-8')` after exec_cmd (my own minor annoyance, feel free to ignore).

Reply:
Yeah, worth cleaning up.